src/realm/table.cpp

```diff
@@ -1548,10 +1495,6 @@ void Table::finalize_migration()
     if (ref) {
         Array::destroy_deep(ref, m_alloc);
         m_top.set(top_position_for_columns, 0);
-        m_top.set(top_position_for_pk_col, 0);
```

Comment:
Why is this no longer needed?

Reply:
Before it was used to indicate that a certain step was completed (convert_links_from_ndx_to_key). This step is now removed.